src/Audio/Output/PlayerAudioSource.cs

```diff
@@ -10,6 +11,7 @@
 using Vintagestory.API.Common;
 using Vintagestory.API.Common.Entities;
 using Vintagestory.API.MathTools;
+using Vintagestory.GameContent;
```

Comment:
PlayerAudioSource shouldn't know about game content, it's not its area of responsibility

Reply:
Done 👍